### PR TITLE
add poetry config for reproducible win build

### PR DESCRIPTION
**Pull request by fkromer**

### CODE REVIEW

File: tt/satisfiability/picosat.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 if os.environ.get('READTHEDOCS') != 'True':
     from tt._clibs import picosat as _c_picosat
-    VERSION = _c_picosat.VERSION
+    VERSION = 965  #_TODO: c_picosat.VERSION
```
Comment (fkromer) [inline tt/satisfiability/picosat.py]: How to fix this?

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: 3.3",
+    "Programming Language :: Python :: 3.4",
+    "Programming Language :: Python :: 3.5",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
```
Comment (fkromer) [inline pyproject.toml]: Build was successful with Python 3.7.6 on Windows 10.

File: setup.py [hunk 3]
```diff
@@ -53,6 +53,7 @@
     'Programming Language :: Python :: 3.4',
     'Programming Language :: Python :: 3.5',
     'Programming Language :: Python :: 3.6',
+    'Programming Language :: Python :: 3.7',
```
Comment (fkromer) [inline setup.py]: Build was successful with Python 3.7.6 on Windows 10.

**Comment by welchbj** [comment]
Looks good! As mentioned in #6, I'll look into the `_c_picosat.VERSION` and clean up the linting issues in the next few days.

**Pull request merged**